ui/pages/destinations/index.js

```diff
@@ -63,26 +67,32 @@ export default function Destinations() {
             cell: info => (
               <div className='flex flex-row items-center py-1'>
                 <div className='mr-3 flex h-9 w-9 flex-none items-center justify-center rounded-md border border-gray-200'>
-                  <img
-                    alt='kubernetes icon'
-                    className='h-5'
-                    src={`/kubernetes.svg`}
-                  />
+                  {info.row.original.kind === 'ssh' ? (
+                    <CommandLineIcon className='h-5 text-black' />
+                  ) : (
+                    <img
+                      alt='kubernetes icon'
+                      className='h-5'
+                      src={`/kubernetes.svg`}
+                    />
+                  )}
                 </div>
                 <div className='flex flex-col'>
                   <div className='text-sm font-medium text-gray-700'>
                     {info.getValue()}
                   </div>
-                  <div className='text-2xs text-gray-500'>
-                    {info.row.original.resources?.length > 0 && (
-                      <span>
-                        {info.row.original.resources?.length}&nbsp;
-                        {info.row.original.resources?.length === 1
-                          ? 'namespace'
-                          : 'namespaces'}
-                      </span>
-                    )}
-                  </div>
+                  {info.row.original.kind !== 'ssh' && (
```

Comment:
For `ssh` let's make the secondary text the url (e.g. `row.original.connection.url`) if it exists. That way users know what hostname to ssh into